### DISCUSSION TITLE
upload to a subfolder path

### DISCUSSION THREAD

**Discussion by m5santovito** [labbots/google-drive-upload#152] [Q&A]
Hi all
I need to upload files to subfolders in Drive. Using the gupload script, I can only upload to a folder in the root, but anytime I provide a subfolder path like "sub1/sub2", I got a new folder sub2 created under root at the same level of sub1, not inside it.
Is there a specific syntax I need to use?
thanks
Maurizio

**Comment by druelie** [comment]
You need to change root to sub2, then the files will be in there

**Reply by m5santovito** [reply]
ok thanks a lot. Is there any way I can get the sub2 id to set the root folder? 
My script is trying to:
1-create sub2 under sub1,
2-set sub2 as root 
3-upload the relevant files under sub2
how can I achieve step 2 in my script?
Thanks a lot in advance

**Reply by druelie** [reply]
As described in the readme, you should be able to do it like:
make sure root is set to sub1
`gupload files -C sub2`

Or use the upload script with the described options.
